pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ClientConfigurationData.java

```diff
@@ -58,6 +58,9 @@
     private int maxLookupRequest = 50000;
     private int maxNumberOfRejectedRequestPerConnection = 50;
     private int keepAliveIntervalSeconds = 30;
+
+    private long connectTimeOutSeconds = 60;
```

Comment:
Can you add two methods in [PulsarAdminBuilder](https://github.com/apache/pulsar/blob/master/pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/PulsarAdminBuilder.java) so people can configure them when constructing the pulsar admin client?

Reply:
Done